### PR TITLE
Made changes to file

### PR DESCRIPTION
**Pull request by Dmambo**
# FEATURES IMPLEMENTED

### Interactions
**Splash screen**

- [x] A simple page with the name of my app and links to the sign up and log in pages.

**Sign up and log in pages**

- [x] The user is  able to register in the app with full name, email and password (all mandatory).
- [x] The user can log into the app using email and password.
- [x] If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).

**Home page (categories page)**

- [x] When the user logs in, they are presented with the categories page.
- [x] For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
- [x] When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
- [x] There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

**Transactions page**

- [x] For a given category, the list of transactions is presented, ordered by the most recent.
- [x] At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
- [x] There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
- [x] When the user clicks on the "Back" button (<), the user navigates to the home page.

**"Add a new category" page**

- [x] The user fills out a form to create a new category, indicating their name and icon (both mandatory).
- [x] The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
- [x] When the user clicks on the "Back" button (<), the user navigates to the home page.

**"Add a new transaction" page**

- [x] The user fills out a form to create a new transaction with:

- name (mandatory)
-  amount (mandatory)
-  categories (mandatory at least one)

- [x] The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
- [x] When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

**Testing requirements**
capybara test

- [ ] Create unit and integration tests for all the most important components .

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Dmambo 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

almost-there

## Suggested changes
_Check the comments under the review._
image

- [ ] Kindly make sure the readme fine does not contain broken links( live demo, presentation link, and the MIT license link.

- [ ] Kindly makes sure you deploy you are App and provide the link in the readme file. The current link is broken 
- [ ] For the documentation requirements I think you need to record a video presentation that contains the following requirements
- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of.


You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -60,15 +60,15 @@ You will create a Ruby on Rails application that allows the user to:
...
+Click to observe the [live project](https://tariq-budget-app.onrender.com/)
 
 <p align="right"><a href="#readme-top">back to top</a></p>
 
 <!-- Live demo -->
 
 ## Presentation <a name="live-presentation"></a>
 
-Click to watch the [presentation on project]()
+Click to watch the [presentation on project](https://www.loom.com/share/e5f4e36b3f754b3389c6cb9e69024917)
```
Comment (Ismailco) [inline README.md]: - [ ] Great work with the app, however, the video presentation doesn't have sound kindly check this out and make sure the video is working properly before using it for the presentation

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @Dmambo, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

giphy


## Highlights

✔️ Great work using active storage
✔️ Great UI
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally


## Suggested changes
- [ ] If you changed your GitHub username and the name of the repo kindly make sure to update the links in your README file, as for now the issues page and LICENSE redirect to `404` page
- [ ] Kindly make sure to add a LICENSE (file) to your repo
_Check the comments under the review._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by BertrandConxy — APPROVED** [review]
Hi @Dmambo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**